papaparse.js

```diff
@@ -1113,6 +1113,10 @@
 						header = header.trim();
 					}
 
+					if (typeof _config.headerTransform === 'function') {
```

Comment:
HeaderTransform should always be a function so instead of silently ignoring the arugment if it is not I prefer to make PapaParse throw and error when it is not a function.